code/game/objects/items.dm

```diff
@@ -1017,6 +1017,11 @@ var/global/list/image/blood_overlays = list()
 	if(flags & INVULNERABLE)
 		return
 	spawn(0) //this is needed or multiple items will be thrown sequentially and not simultaneously
+		if(anchored)
```

Comment:
Yeah, singularities move things that are anchored, and unanchor them
Great fun with a mech